src/app/layouts/navbar.component.ts

```diff
@@ -1,12 +1,15 @@
 import { Component, OnInit } from "@angular/core";
 
 import { MessagingService } from "jslib-common/abstractions/messaging.service";
+import { OrganizationService } from "jslib-common/abstractions/organization.service";
 import { PlatformUtilsService } from "jslib-common/abstractions/platformUtils.service";
 import { ProviderService } from "jslib-common/abstractions/provider.service";
 import { SyncService } from "jslib-common/abstractions/sync.service";
 import { TokenService } from "jslib-common/abstractions/token.service";
 import { Provider } from "jslib-common/models/domain/provider";
 
+import { NavigationPermissionsService as OrgNavigationPermissionsService } from "../organizations/services/navigation-permissions.service";
```

Comment:
What do you think of just naming that service `OrganizationNavigationPermissionsService` to begin with, and get rid of these aliasing patterns?

Reply:
I also thought about this, BUT all aliasing patterns will be dropped once I introduce the `OrganizationModule`. Most things are used within that module and won't need to be prefixed with `Org`. So this is just temporary.